scripts/setup/configure-certs

```diff
@@ -0,0 +1,24 @@
+#!/bin/env bash
+
+# Realm-specific construct
+if [ -z "$RLMNAME" ]; then
+    echo "No Realm name given."
+    RLMNAME="A boson of realm"
+fi
+
+echo "Executing certificates configuration..."
+if [ ! -e /etc/ssl/private/zulip.key ] && [ ! -e /etc/ssl/certs/zulip.combined-chain.crt ]; then
+    echo "SSL certificates for Zulip not found in /etc/ssl/."
+    echo "Autogenerating certificates ..."
+    sudo -i
```

Comment:
I think this script should expect to be run as root rather than using sudo.  `/etc/ssl/private` isn't world-readable usually.